### DISCUSSION TITLE
PlugData

### DISCUSSION THREAD

**Discussion by GSNinjaMusic** [catppuccin/catppuccin#2387] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://github.com/plugdata-team/plugdata

![327259409-32d8dd9a-7644-49ed-a759-1db4c6f2b295](https://github.com/catppuccin/catppuccin/assets/84602064/43b6454c-81b1-469c-9383-494415596dd4)

![327259496-7bee4232-3b4b-4e23-9239-db079bf917e3](https://github.com/catppuccin/catppuccin/assets/84602064/51e7f6e9-ec10-4069-bd8f-304e5d96c948)

![327259520-5d739dda-54e1-45e9-b3fd-3b754045689e](https://github.com/catppuccin/catppuccin/assets/84602064/5e540e96-a1e7-4504-9340-1e845de49159)

![327259562-46cef351-3f36-4460-855b-0f94f4ee1071](https://github.com/catppuccin/catppuccin/assets/84602064/6ad72c14-053d-4952-91a7-bf15ef8427e8)

![327259586-cb80eb72-2efb-4b26-96ac-e61ee1402d41](https://github.com/catppuccin/catppuccin/assets/84602064/91c0ea5a-5ba1-483e-b771-937333770617)

### Any additional comments?

These prints are what you need to edit the application theme

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by scarcekoi** [comment]
Done: #2970 & #2975.

**Reply by GSNinjaMusic** [reply]
THANK YOU VERY MUCH

**Reply by scarcekoi** [reply]
Your welcome!

**Comment by scarcekoi** [comment]
Official port now exists: https://github.com/catppuccin/plugdata
